zebra-rpc/src/methods.rs

```
@@ -357,7 +353,9 @@ where
 
             // send transaction to the rpc queue
             let unmined_transaction = UnminedTx::from(raw_transaction.clone());
-            let _ = queue_sender.send(Some(unmined_transaction)).await;
+            queue_sender
+                .send(Some(unmined_transaction))
+                .expect("there is always at least one active receiver");
```

Question: Could this `expect()` cause shutdown panics?

---

We want to ignore this error, because there is nothing useful we can do about it.

---

Done as part of https://github.com/ZcashFoundation/zebra/pull/4015/commits/f82528104bd9b0c8573f196958f61cdd43a6652b